src/VaultSharp/V1/Commons/SubKeysInfo.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using VaultSharp.Core;
+
+namespace VaultSharp.V1.Commons
+{
+    public class SubKeysInfo
+    {
+        [JsonProperty("subkeys")]
+        public IDictionary<string, object> SubKeys { get; set; }
```

Comment:
If we change this to concrete Dictionary, you can get rid of OnDeserialized etc. right?